xbmc/addons/interfaces/General.h

```diff
@@ -52,10 +52,10 @@ namespace ADDON
     static char* get_region(void* kodiBase, const char* id);
     static void get_free_mem(void* kodiInstance, long* free, long* total, bool as_bytes);
     static int  get_global_idle_time(void* kodiBase);
+    static void kodi_version(void* kodiBase, char** compile_name, int* major, int* minor, char** revision, char** tag, char** tagversion);
```

Comment:
This is the wrong place.

Reply:
This to bring backward compatibility

Reply:
I know, but `kodi_version` was after `get_current_skin_id` in this struct.

Reply:
No was before https://github.com/xbmc/xbmc/blob/4398f988e1577495ab6a9ba1e43e6bd2ac65f465/xbmc/addons/kodi-addon-dev-kit/include/kodi/General.h#L37-L38 the `get_current_skin_id` was in newer versions added, thats why behind.

Reply:
You looked at / linked to the wrong file, this is `xbmc/addons/interfaces/General.h` not `xbmc/addons/kodi-addon-dev-kit/include/kodi/General.h`.

Reply:
True, mix these more often in my head. But think of sorting it right there in a dishwashing, not bad.

Reply:
> But think of sorting it right there in a dishwashing, not bad

 Yes, but can we please make this change later, in another PR?